server/src/test/java/org/elasticsearch/cluster/metadata/DateMathExpressionResolverTests.java

```diff
@@ -108,20 +108,20 @@ public void testExpression_EscapeStatic() throws Exception {
 
     @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/37037")
```

Comment:
The awaitsFix that were added due to these failures also need to be removed. Otherwise such tests won't be run, and the fix won't matter :)